ios/objective-c/AmpliObjectiveCSampleApp/AmpliObjectiveCSampleApp/Ampli/Ampli.m

```diff
@@ -674,7 +677,9 @@ - (void)identify:(NSString *_Nullable)userId event:(Identify *)event options:(Ev
     if (![self isInitializedAndEnabled]) {
         return;
     }
-    if (userId != nil) {
+    if (options != nil && options.userId != nil) {
```

Comment:
LGTM but does it help in our case?
I think, here
![image](https://user-images.githubusercontent.com/9509929/225659994-e2fecf77-5bba-455b-aba3-f346cdc27279.png)
`b.userId` is a property added by the customer , not a property of `EventOptions`

I think, possible solution is to re-write customer's function
![image](https://user-images.githubusercontent.com/9509929/225660798-d82347b5-0adb-4100-add9-7f14b5aad47f.png)
to
```
Ampli.instance().identify(userProperty.userId, event: userProperty)
```

Reply:
Thanks Andrey, 100% agree but wanted to fix the EventOptions support before responding so that I could tell them they can set userId and deviceId via EventOptions and to use that instead of clouding up the Identify user_properties.